R/create_episode_file.R

```diff
@@ -428,3 +421,57 @@ join_cohort_lookups <- function(
 
   return(join_cohort_lookups)
 }
+
+
+#' Join sc client variables onto episode file
+#'
+#' @description Match on sc client variables.
+#'
+#' @param individual_file the processed individual file
+#' @param year financial year.
+#' @param sc_client SC client lookup
+#' @param sc_demographics SC Demographic lookup
+join_sc_client_ep <- function(episode_file,
+                              year,
+                              sc_client = read_file(get_sc_client_lookup_path(year)),
+                              sc_demographics = read_file(
+                                get_sc_demog_lookup_path(),
+                                col_select = c("sending_location", "social_care_id", "chi")
+                              )) {
+  # Match to demographics lookup to get CHI
+  client_count_not_known <-
+    read_file(get_sc_client_lookup_path(year)) %>%
```

Comment:
```suggestion
    sc_client %>%
```
this was also specified in the parameter so can be simlified